src/components/Form.tsx

```diff
@@ -144,11 +260,17 @@ export const FormWithRows: React.FC<{formDetails: FormDescriptionRow}> = (props)
                                 // TODO add support for input type password
                                 if (field.type === FormElementType.passwordGroup) {
                                     const inputField = field as PasswordGroupField
-                                    return  <div style={{display: "flex", width: "67%"}}>
+                                    return  <div className="passwordAndHelperText">
+                                                <div className="passwordGroup"> 
                                                 <TextField
+                                                error={errorsPassword.length > 0 ? true : false}
```

Comment:
you can also just use `errorsPassword.length > 0` since it is the same as the conditions you have.

Reply:
okay :)